content/issues/2023-09-04-artifactory-central-brownount.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Brownout of the Maven Central mirrored repositories in JFrog Artifactory (repo.jenkins-ci.org)
+date: 2023-09-04T12:00:00-00:00
+resolved: false
+resolvedWhen: 2023-09-04T16:00:00-00:00
+# Possible severity levels: down, disrupted, notice
+severity: notice
+affected:
+  - repo.jenkins-ci.org
+  - ci.jenkins.io
+  - infra.ci.jenkins.io
+section: issue
+---
+
+<!-->
+[Final Message]
+Brownout is finished after 1h30 (instead of initial 1 hour).
+
+[Initial Message]
+-->
+
+Monday 04 of September 2023, we'll perform a [brownout](https://en.wikipedia.org/wiki/Brownout_(software_engineering)) operation of 4 hours on the Jenkins Artifactory service at repo.jenkins-ci.org at 10:00 UTC.
+
+This operation consists in removing all mirrors of the [Maven Central](https://search.maven.org/) artifact repository from repo.jenkins-ci.org to force Maven builds to download mirrored artefacts from the Maven central instead of our Artifactory instance.
```

Comment:
I assume it is intentional that you didn't include the extra text in the URL that would avoid the invitation to redirect to sonatype.  I think that your original choice is probably the better choice as a way to honor Sonatype's hosting of maven central.  However, I offer this suggestion as entirely optional in case you'd like to avoid the redirect message.
```suggestion
This operation consists in removing all mirrors of the [Maven Central](https://search.maven.org/?eh=) artifact repository from repo.jenkins-ci.org to force Maven builds to download mirrored artefacts from the Maven central instead of our Artifactory instance.
```